src/modelstats.jl

```diff
@@ -3,47 +3,18 @@ export dic
 #################### Posterior Statistics ####################
 
 """
-    dic(chain::Chains, logpdf::Function) -> (DIC, pD)
+    dic(chain::Chains)
 
-Compute the deviance information criterion (DIC).
-(Smaller is better)
+Computes Deviance Information Criterion (DIC).
 
-Note: DIC assumes that the posterior distribution is approx. multivariate Gaussian and tends to select overfitted models.
-
-## Returns:
-* `DIC`: The calculated deviance information criterion
-* `pD`: The effective number of parameters
-
-## Usage:
-
-```
-chn ... # sampling results
-lpfun = function f(chain::Chains) # function to compute the logpdf values
-    niter, nparams, nchains = size(chain)
-    lp = zeros(niter + nchains) # resulting logpdf values
-    for i = 1:nparams
-        lp += map(p -> logpdf( ... , x), Array(chain[:,i,:]))
-    end
-    return lp
-end
-
-DIC, pD = dic(chn, lpfun)
-```
+# Arguments
```

Comment:
This seems to be useful information, maybe it should be added to StatsModelComparisons (as well)?
```suggestion
Note: DIC assumes that the posterior distribution is approx. multivariate Gaussian and tends to select overfitted models.

# Arguments
```